ophyd/pv_positioner.py

```diff
@@ -149,10 +149,12 @@ def moving(self):
     def _setup_move(self, position):
         '''Move and do not wait until motion is complete (asynchronous)'''
         self.log.debug('%s.setpoint = %s', self.name, position)
-        self.setpoint.put(position, wait=True)
         if self.actuate is not None:
+            self.setpoint.put(position, wait=True)
```

Comment:
What there is an `actuate` and setpoint has put completion? Seems like a blanket `wait=False` is what you're after.

But then it gets back to the original point of "choose the right class, please".  I'm really not sold on the argument that it's our responsibility to patch it here. 

The idea of keeping `setpoint.put(..., wait=True)` in both instances is that it would block while allowing the IOC to go through whatever bizarre process it needs to update the setpoint or re-initialize other status PVs. Setting `wait=False` could potentially get in the way of what may be a race to check motion status.

Reply:
That is convincing. I will close this and see if `PVPositioner` -> `PVPositionerPC` fixes the issue at the beamline. I may reopen discussion if we find that we don't fully understand the issue, but I think that we do.